pkg/queue/stats_test.go

```diff
@@ -69,9 +68,9 @@ func TestSingleRequestHalfTime(t *testing.T) {
 	s := newTestStats(now)
 
 	s.requestStart(now)
-	now = now.Add(1 * time.Second)
+	now = now.Add(500 * time.Millisecond)
 	s.requestEnd(now)
-	now = now.Add(1 * time.Second)
+	now = now.Add(500 * time.Millisecond)
```

Comment:
So before we were able to work with random intervals, now they have to be precise 1s?

Reply:
It sounds not right.

Reply:
They do not, the test hardcodes that currently. Will add tests with diff intervals.